lib/galaxy/webapps/galaxy/api/libraries.py

```diff
@@ -239,7 +240,7 @@ def get_permissions( self, trans, encoded_library_id, **kwd ):
             raise exceptions.RequestParameterInvalidException( "The value of 'scope' parameter is invalid. Alllowed values: current, available" )
 
     @expose_api
-    def set_permissions( self, trans, encoded_library_id, **kwd ):
+    def set_permissions( self, trans, encoded_library_id, payload=None, **kwd ):
         """
         def set_permissions( self, trans, encoded_dataset_id, **kwd ):
```

Comment:
This doc line is not in sync with the method signature, I'd suggest to remove it.